test/export-default/fixtures/class.esm.advanced.js

```diff
@@ -1 +1 @@
-export default class{constructor(b){this.a=b}console(){console.log(this.a)}}
+class a{constructor(b){this.a=b}console(){console.log(this.a)}}export default a;
```

Comment:
Slight regression in size here, since we are no longer exporting the default defined class, but we can likely recover the delta in a future enhancement.